node/Dockerfile.amd64

```diff
@@ -219,6 +219,7 @@ RUN /clean-up-filesystem.sh
 # Copy everything into a fresh scratch image so that naive CVE scanners don't pick up binaries and libraries
 # that have been removed in our later layers.
 FROM scratch
+# base ubi image has been cleaned from  clean-up-filesystem.sh of all unecessary imports
```

Comment:
I think this comment isn't needed, it's explained in the two comments above.

Reply:
At a minimum, should clean this up so that it starts with a capital letter "Base" and remove the extra space between "from" and "clean-up-filesystem" (there are two)

Reply:
https://github.com/projectcalico/calico/pull/6284/commits/5b19f25c502f9a6d308738d726ec285b4e899199 - removed